### PR TITLE
arm64, x86: Update bindings to kernel v6.0

### PR DESCRIPTION
**Pull request by rbradford**
Signed-off-by: Rob Bradford <robert.bradford@intel.com>

### CODE REVIEW

**Comment by lauralt** [comment]
we should probably update the CONTRIBUTING file as well.

File: src/arm64/bindings.rs [hunk 1]
```diff
@@ -3820,304 +4106,531 @@ fn bindgen_test_layout_kvm_run__bindgen_ty_1__bindgen_ty_12() {
     );
 }
 #[repr(C)]
-#[derive(Debug, Default, Copy, Clone, PartialEq)]
+#[derive(Copy, Clone)]
```
Comment (lauralt) [inline src/arm64/bindings.rs:4109]: I think we had this problem in virtio-bindings as well and the decision there was to manually implement the traits that are removed in the new bindings. @rbradford @sboeuf @andreeaflorescu what it's your suggestion here?

Comment (andreeaflorescu) [reply src/arm64/bindings.rs:4109]: I would say to add them manually to avoid breaking changes.

**Comment by rbradford** [comment]
Superseded by #82

**Pull request closed without merging**